service-api/src/main/java/greencity/dto/order/EditPriceOfOrder.java

```diff
@@ -13,9 +17,9 @@
 public class EditPriceOfOrder {
     @NotNull
     @Min(0)
-    private Long minPriceOfOrder;
+    private Long minQuantity;
```

Comment:
what does `minQuantity` means here
I see it is `EditPriceOfOrder` class and according to the name it should be responsible for the `price` of the `order` not of the `amount` of orders
So `minPriceOfOrder` is better name for the minimum price of the order

Reply:
This DTO is also used in the `SuperAdminController` 
endpoint ` @PatchMapping("/setLimitsBySumOfOrder/{tariffId}")`
Is `UI` also changed to use new names?

Reply:
at finally I'll discuss it 
Before the merging I'll notice to Vika about that changes and also notice to UI part to make that changes using those field